test/infrastructure/container/interface.go

```diff
@@ -95,6 +97,8 @@ type RunContainerInput struct {
 	PortMappings []PortMapping
 	// IPFamily is the IP version to use.
 	IPFamily clusterv1.ClusterIPFamily
+	// Resource limits and settings for the container.
+	Resources dockercontainer.Resources
```

Comment:
This could also be exposed at the DockerCluster level for resource limits on the load balancer, but I wanted some feedback before making this a user-facing change.